### DISCUSSION TITLE
Saving different camera positions/zoom for quick access to zones

### DISCUSSION THREAD

**Discussion by Mikerhinos** [Comfy-Org/ComfyUI#1197] [Ideas]
With this many features like SDXL base, refiner, highres fix, ultimate SD upscale, face detectors etc. we quickly have huge workflows.
I find it would be cool to have a feature in the floating menu to save current camera position and zoom on the workflow and be able to save as many as we want to get some shortcuts to it.
Like I have a shortcut for a part of the workflow with all the prompts, one for a part with the previews, etc. Any shortcut with a precise position and zoom.

**Comment by donengel72** [comment]
Interesting idea, actually as my nodes get bigger and bigger.

**Comment by Mikerhinos** [comment]
Closing this discussion as this solution already exists : graphNavigator by rock-land .
https://github.com/rock-land/graphNavigator
Awesome !
